test/test.js

```diff
@@ -79,5 +80,24 @@ describe('Serializer', () => {
 
       strictEqual(toCanonical(result), toCanonical(quads))
     })
+
+    describe('should generate module when the type is', () => {
+      [undefined, 'commonjs', 'esm', 'ts'].forEach(module => {
```

Comment:
Maybe move the data building step to `support/example.js` and create separate `it()` calls for each tests? Would be more readable. Since this is not a spec test, I think it would match better.